### PR TITLE
Add Travis OSX CI

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

**Comment by tribal-tec** [comment]
Retest this please

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+notifications:
+  email:
+    on_success: never
+language: cpp
+sudo: false
+os:
+  - osx
```
Comment (rolandjitsu) [inline .travis.yml]: Quick question, any reason why an OS X image?

Comment (tribal-tec) [reply .travis.yml]: The linux builds are done by Jenkins, and we offload the OSX builds on Travis

**Comment by tribal-tec** [comment]
The webhooks don't seem to work yet...

File: .travis.yml (hunk 1 above)
Comment (rolandjitsu) [reply .travis.yml]: Got it 😄

**Comment by tribal-tec** [comment]
Retest this please

**Comment by tribal-tec** [comment]
Ha! Added bbpbuildbot to collaborators, now the webhook seems to work.

**Comment by rdumusc** [comment]
Ah yes, that's what was missing..!
I'll rebase this on your PR once it's merged for the CI build to pass.

**Comment by rdumusc** [comment]
retest this please

**Pull request merged**